test/check-machines-migrate

```diff
@@ -163,7 +163,7 @@ class TestMachinesMigration(VirtualMachinesCase):
 
         b.set_checked("#temporary", temporary)
 
-        if self.machine.image in ["rhel-8-6", "rhel-9-0", "rhel-9-1", "centos-9-stream"]:
+        if self.machine.image.startswith("rhel") or self.machine.image.startswith("centos"):
```

Comment:
This looked intuitively correct -- why would it be disabled on rhel 8 and centos 9, but work on centos 8? But that's [exactly what happens](https://logs.cockpit-project.org/logs/pull-708-20220516-132615-c6bcc969-centos-8-stream/log.html#61)... So, fixing this to be "centos-9" again.